### ISSUE TITLE
Implement Accessibility Testing in Playwright Suite

### ISSUE THREAD

**Issue by sneha-sirure** [Altinn/altinn-access-management-frontend#1204]
**Integrate automated accessibility checks into Playwright test suite to identify and address accessibility issues early in the development process**

- [x] Set Up Required Packages : 
  - [x] Install Axe-Core for Accessibility Testing
  - [x] Import Necessary Libraries in Your Test Files
- [x] Identify the Pages and Components for Accessibility Testing
- [x] Write Accessibility Test for Enkeltjenestedelegering for new brukerflate GUI 
- [x] Run these tests to catch any accessibility violations early

**Comment by sneha-sirure** [comment]
@Siri-Anna Lukker tasken siden den er ferdig, men ikke pushet kode enda pga den skal gi feil test i github actions, men finner en løsning hvis vi kunne kjøre testene seperate i github actions.
